### PR TITLE
add make target for metricsbridge

### PR DESCRIPTION
**Pull request by mjudeikis**
@kargakis 
/hold
need to sync with release repo PR https://github.com/openshift/release/pull/2279

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,13 @@
 COMMIT=$(shell git rev-parse --short HEAD)$(shell [[ $$(git status --porcelain --ignored) = "" ]] && echo -clean || echo -dirty)
 
 # all is the default target to build everything
-all: clean build azure-controllers etcdbackup sync
+all: clean build azure-controllers etcdbackup sync metricsbridge
```
Comment (kargakis) [inline Makefile]: Can't see the new target anywhere in the Makefile

Comment (kargakis) [reply Makefile]: Hrm, seems to be there it's just your master branch is old.

/lgtm

Comment (mjudeikis) [reply Makefile]: github cache 
https://github.com/openshift/openshift-azure/blob/master/Makefile#L44

**Comment by mjudeikis** [comment]
@kargakis I want to merge https://github.com/openshift/openshift-azure/pull/779 and rebase first as it will clash

**Pull request merged**